packages/admin-next/dashboard/src/i18n/translations/en.json

```diff
@@ -336,6 +335,12 @@
         "successToast": "Option {{title}} was successfully created."
       }
     },
+    "organization": {
```

Comment:
this breaks organization section title

![Screenshot 2024-06-04 at 15 45 39](https://github.com/medusajs/medusa/assets/16856471/f157fc36-ec7b-4cf6-a28a-5d22fb83f51f)